azurestack/resource_arm_loadbalancer_backend_address_pool.go

```diff
@@ -0,0 +1,243 @@
+package azurestack
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2015-06-15/network"
+	"github.com/hashicorp/errwrap"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurestack/azurestack/utils"
+)
+
+func resourceArmLoadBalancerBackendAddressPool() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmLoadBalancerBackendAddressPoolCreate,
+		Read:   resourceArmLoadBalancerBackendAddressPoolRead,
+		Delete: resourceArmLoadBalancerBackendAddressPoolDelete,
+		Importer: &schema.ResourceImporter{
+			State: loadBalancerSubResourceStateImporter,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"location": deprecatedLocationSchema(),
```

Comment:
As this is a new resource it might make more sense to remove `location` instead of deprecating it?

Reply:
makes sense, will remove it from now on.